app/webpack.common.js

```diff
@@ -45,9 +45,10 @@ const commonConfig = {
       },
       {
         test: /\.node$/,
-        use: [
-          { loader: 'node-native-loader', options: { name: '[name].[ext]' } },
```

Comment:
Running this in the scope of webpack 4 gives us this error whenever we touch a native node module: `TypeError: Cannot read property 'nodeNativeLoader' of undefined`. I couldn't see any indication this was updated to support webpack 4, so I churned us over to something compatible.